oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -1101,12 +1111,16 @@ impl Runtime {
 
     /// Update the node count metric with the current value.
     fn update_nodes_count_metric(&self) {
-        METRICS.runtime_nodes_count.set(
-            self.node_infos
-                .read()
-                .expect("could not acquire lock on node_infos")
-                .len() as i64,
-        );
+        self.metrics_data
+            .runtime_metrics
+            .runtime_nodes_total
+            .with_label_values(&["runtime"])
+            .set(
+                self.node_infos
+                    .read()
+                    .expect("could not acquire lock on node_infos")
+                    .len() as i64,
```

Comment:
Maybe refactor this in a separate method, to make it clear that it does not hold any locks.

Reply:
Thanks. Done!